app/style.css

```diff
@@ -33,17 +33,28 @@
 	border: solid 4px;
 }
 
-.instructionsContainer {
+.gameHeader {
 	height: 20vh;
```

Comment:
So your JS is indented with two spaces, but CSS with four?

Reply:
really awkward, don't see it in my local file

Reply:
It's probably because you're using tabs in CSS. In GitHub, tabs are displayed as 4 spaces, while in your editor you probably see it as two.

Reply:
indeed.

think it will be fixed in the next commit.